### PR TITLE
Public Cloud - vRack - New tutorials: SNS & Ubika WAAP Gateway

### PR DESCRIPTION
**Pull request by Y0Coss**
Replaces #6895

### CODE REVIEW

**Comment by fabienOVH** [comment]
Fr versions of Ubika and Stormshield done

File: pages/public_cloud/public_cloud_network_services/tutorial-stormshield_network_security_vrack/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,560 @@
...
+>
+> Veuillez noter que toutes les rubriques relatives à la « Haute disponibilité » ou à « Stormshield-2 » sont facultatives, de même que l'utilisation du réseau vRack avec Additional IP. Ils sont inclus pour montrer comment mettre en place le système avec deux instances en mode actif/passif pour une haute disponibilité. Dans une version minimale, il peut également fonctionner avec une seule instance si cela suffit à vos besoins.
+
+> [!primary]
+> Dans ce scénario, nous utiliserons deux machines virtuelles configurées pour l'appliance de sécurité afin d'atteindre la haute disponibilité (**H**igh **A**vailability ou HA) et une machine virtuelle supplémentaire pour l'administration. Cette configuration assure une protection du failover et une disponibilité continue du service. Pour plus d'exemples et de conseils détaillés sur les options d'évolutivité, veuillez consulter la [documentation de Stormshield](https://documentation.stormshield.eu/HOME/Content/Website_Topics/Root-HomePage-FR.htm){.external}.
```
Comment (Y0Coss) [inline pages/public_cloud/public_cloud_network_services/tutorial-stormshield_network_security_vrack/guide.fr-fr.md]: Original EN version: "This setup ensures failover protection"
Your original FR translation @fabienOVH: "Cette configuration assure une protection contre le basculement"
My version: "Cette configuration assure une protection du failover"

I'm not convinced my version is the appropriate one but translating the EN to FR is tricky. Can you check please?

Comment (Y0Coss) [reply pages/public_cloud/public_cloud_network_services/tutorial-stormshield_network_security_vrack/guide.fr-fr.md]: Same question for the Ubika guide BTW

File: pages/public_cloud/public_cloud_network_services/tutorial-stormshield_network_security_vrack/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,560 @@
...
+
+```bash
+openstack subnet create --network stormshield-ha --subnet-range 192.168.2.0/29 --dhcp --gateway none stormshield-ha
+```
+
+#### Déployer les instances SNS
+
+Rendez-vous dans la section `download` du [site officiel de Stormshield](https://documentation.stormshield.eu/SNS/v4/fr/Content/PAYG_Deployment_Guide/Downloading_installation_file.htm){.external}. Connectez-vous à votre compte Stormshield et suivez les instructions pour télécharger l'image Stormshield OpenStack.
+
+Rendez-vous dans le dossier où vous avez téléchargé votre image SNS Openstack et téléchargez l'image (pour ce tutoriel, nous utilisons l'image `utm-SNS-EVA-4.8.3-openstack.qcow2`) :
```
Comment (Y0Coss) [inline pages/public_cloud/public_cloud_network_services/tutorial-stormshield_network_security_vrack/guide.fr-fr.md]:
```suggestion
Rendez-vous dans le dossier où vous avez téléchargé votre image SNS Openstack et téléversez l'image (pour ce tutoriel, nous utilisons l'image `utm-SNS-EVA-4.8.3-openstack.qcow2`) :
```
EN guide mentions upload, not download. Can you check please?

File: pages/public_cloud/public_cloud_network_services/tutorial-ubika_vrack/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+openstack router add subnet 2481bcaf-efa2-419a-ad92-d6d27737dfd1 ubika-workload
+```
+
+#### Déployer les instances Ubika WAAP Gateway
+
+Téléversez l'image UWG image vers OpenStack:
+
+Rendez-vous dans la section `download`{.action} du [site officiel d'Ubika](https://my.ubikasec.com/){.external}. Connectez-vous à votre compte Ubika et suivez les instructions pour télécharger l'image UWG OpenStack.
+
+Rendez-vous dans le dossier où vous avez téléchargé votre image UWG OpenStack et téléversez l'image UWG OpenStack (pour ce tutoriel, nous utilisons l'image `UBIKA_WAAP_Gateway-generic-cloud-6.11.10+51a56f6201.b56855.qcow2`) :
```
Comment (Y0Coss) [inline pages/public_cloud/public_cloud_network_services/tutorial-ubika_vrack/guide.fr-fr.md]: I chose to translate closely the "download/upload" terms from the EN version. But this leads to this step now looking a bit weird. Can you check if this is appropriate? (EN/FR)

File: pages/public_cloud/public_cloud_network_services/tutorial-stormshield_network_security_vrack/guide.fr-fr.md (hunk 1 above)
Comment (Y0Coss) [reply pages/public_cloud/public_cloud_network_services/tutorial-stormshield_network_security_vrack/guide.fr-fr.md]: Fixed with https://github.com/ovh/docs/pull/7064/commits/bf8431c197d37cda38568d8c537b1faaedff3e30
Thanks!

File: pages/public_cloud/public_cloud_network_services/tutorial-stormshield_network_security_vrack/guide.fr-fr.md (hunk 2 above)
Comment (fabienOVH) [reply pages/public_cloud/public_cloud_network_services/tutorial-stormshield_network_security_vrack/guide.fr-fr.md]: Yes "téléversez" or "importez" is correct

File: pages/public_cloud/public_cloud_network_services/tutorial-ubika_vrack/guide.fr-fr.md (hunk 3 above)
Comment (fabienOVH) [reply pages/public_cloud/public_cloud_network_services/tutorial-ubika_vrack/guide.fr-fr.md]: for me the translation is okay

**Comment by Y0Coss** [comment]
Thank you for your contribution @fabienOVH

**Pull request merged**